clients/src/main/resources/common/message/StreamsPrepareAssignmentResponse.json

```diff
@@ -0,0 +1,105 @@
+  "type": "response",
+  "name": "StreamsPrepareAssignmentResponse",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  // Supported errors:
+  // - GROUP_AUTHORIZATION_FAILED
+  // - NOT_COORDINATOR
+  // - COORDINATOR_NOT_AVAILABLE
+  // - COORDINATOR_LOAD_IN_PROGRESS
+  // - INVALID_REQUEST
+  // - INVALID_GROUP_ID
+  // - GROUP_ID_NOT_FOUND
+  // - UNKNOWN_MEMBER_ID
+  // - STALE_MEMBER_EPOCH
+  "fields": [
+    { "name": "ThrottleTimeMs", "type": "int32", "versions": "0+",
+      "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
+    { "name": "ErrorCode", "type": "int16", "versions": "0+",
+      "about": "The top-level error code, or 0 if there was no error" },
+    { "name": "ErrorMessage", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
+      "about": "The top-level error message, or null if there was no error." },
+    { "name": "GroupEpoch", "type": "int32", "versions": "0+",
+      "about": "The group epoch." },
+    { "name": "AssignorName", "type": "string", "versions": "0+",
+      "about": "The selected assignor." },
+    { "name": "Members", "type": "[]Member", "versions": "0+",
+      "about": "The members.", "fields": [
+      { "name": "MemberId", "type": "string", "versions": "0+",
+        "about": "The member ID." },
+      { "name": "MemberEpoch", "type": "int32", "versions": "0+",
+        "about": "The member epoch." },
+      { "name": "InstanceId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
+        "about": "The member instance ID." },
+      { "name": "RackId", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
+        "about": "The rack ID." },
+
+      { "name": "TopologyHash", "type": "bytes", "versions": "0+",
+        "about": "The hash of the topology." },
+
+      { "name": "ActiveTasks", "type": "[]TaskId", "versions": "0+",
+        "about": "Target active tasks for this consumer." },
+      { "name": "StandbyTasks", "type": "[]TaskId", "versions": "0+",
+        "about": "Target standby tasks for this consumer." },
+      { "name": "WarmupTasks", "type": "[]TaskId", "versions": "0+",
+        "about": "Target warming up tasks for this consumer. " },
+
+      { "name": "ProcessId", "type": "uuid", "versions": "0+",
+        "about": "Identity of the streams instance that may have multiple consumers. " },
+      { "name": "HostInfo", "type": "HostInfo", "versions": "0+",
+        "about": "Host information for running interactive queries on this instance. ",
+        "fields": [
+          { "name": "Host", "type": "string", "versions": "0+",
+            "about": "Host for running interactive queries on this instance." },
+          { "name": "Port", "type": "int32", "versions": "0+",
+            "about": "Port for running interactive queries on this instance." }
+        ]
+      },
```

Comment:
Do we need this in the response? This information is distributed to the members for allowing IQ, I do not think it is needed to compute the assignment.

Reply:
You are right we probably won't need this.